.github/workflows/test.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         gcloud firebase test android run --type instrumentation \
         --app android/app/build/outputs/apk/debug/app-debug.apk \
         --test android/app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk \
-        --device model=MediumPhone.arm,version=33,locale=en,orientation=portrait \
```

Comment:
maybe you could revert `MediumPhone` in order to have a more generic device (those devices are encouraged by google)